src/App.css

```diff
@@ -42,3 +42,127 @@
 .social-links a:hover {
   text-decoration: underline;
 }
+
+section {  
+  margin: auto;
+}
+
+.team {  
+  padding: 0 1rem;
+  margin: 4rem auto 4rem auto;
+  max-width: 970px;
+  min-width: 20rem;  
+}
+
+.team-header h2 {
+  color: var(--color-grey-dark);
+  font-weight: 300;
+  font-size: 2rem;
+}
+
+.team-body {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(18rem, 1fr));
+  gap: 1rem;
+}
+
+.team-member-card {
+  aspect-ratio: var(--ratio-landscape);
+  box-shadow: var(--shadow-2);
+  border-radius: 1.5rem;
+  display: flex;
+  justify-content: center; 
+  align-items: center;
+  transition: all .3s ease-in-out;
+  background-image: linear-gradient(0deg,var(--color-yellow),white); 
+  background-position: 0 -100%; 
+  background-repeat: no-repeat; 
+  background-size: 100% 200%;  
+  translate: 0; 
+  animation: var(--animation-fade-in);
+}
+
+.team-member-card a {
+  display: flex;
+  justify-content: space-around;    
+  align-items: center;
+  text-decoration: none;
+  width: 100%;  
+  height: 100%;
+  color: #000;
+  font-size: var(--font-size-1);
+}
+
+.team-member-profile {
+  display: flex;
+  flex-direction: column;  
+  align-items: center;  
+  justify-content: space-evenly;
+  gap: 0.1rem;
+}
+
+.team-member-card .avatar {
+  border-radius: 50%;
+  flex-shrink: 0;
+  width: 6rem;
+  height: 6rem;
+  transition: scale .3s ease-in-out;
+  scale: 1;
+  margin-right: 0;  
+}
+
+.team-member-card:hover {
+  outline-color: var(--color-red);
+  background-position: 0 -50%;
+  translate: 0 -.2rem; 
+  box-shadow: var(--shadow-3);
+}
+
+.team-member-card:hover .avatar { 
+  scale: 1.1;  
+} 
+
+.team-member-card-text {
+  display: flex;
+  flex-direction: column;  
+  align-items: center;
+}
+
+.team-member-card-heading {
+  margin-bottom: 0;  
+  font-size: 1.1rem;
+  font-weight: var(--font-weight-7);
+  transition: color .4s ease-in-out;
+}
+
+.team-member-card:hover .team-member-card-heading {  
+  color: var(--color-red);
+}
+
+.team-member-card:hover:after, .team-member-card:focus-visible:after { 
+  opacity: 1; 
+  inset: auto auto .5rem;
+} 
+
+.team-member-card span {  
+  text-decoration: none;
+  color: var(--color-grey);
+  position: relative;  
+}
+
+.team-member-card span::after {  
+  content: '';
+  position: absolute;
+  transition: width .3s ease-in-out;
+  bottom: 0;
+  left: 50%;
+  width: 0;
+  height: 1px;
+  background-color:var(--color-grey);  
+  transform: translateX(-50%);
+}
+
+.team-member-card:hover span::after {
+  width: 100%;
+  left: 50%;
+}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** Transitioning `width` and `height` force the browser to do expensive layout and content flow calculations‏, even if the element doesn't affect the parent.

As the intended effect is to change the background color, using a `linear-gradient` and changing the `background-size` property will work, as well as the `transform` property with a `scale` value.

Those properties will only affect the paint process and are GPU accelerated.

Here is a reference of which property triggers what: https://css-triggers.com/

In order of cheap to expensive: paint > layout > composite